src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -306,10 +306,26 @@ cdef class Blob:
         inst._hb_blob = blob
         return inst
 
+    @classmethod
+    def from_face(cls, face: Face) -> Blob:
```

Comment:
That is an odd place to have it, it should be a  Face method.

Reply:
Oh, hm, okay. The purpose is to get access to a blob to write to disk. What's the best way?